instruqt-tracks/terraform-cloud-aws-v2/02-terraform-cloud-setup/check-workstation

```diff
@@ -91,27 +91,30 @@ STATUS=$(curl \
     -w "%{response_code}" \
     -s \
     -o /tmp/.out.json \
-    https://app.terraform.io/api/v2/organizations/"${ORG}"/entitlement-set 2>/dev/null) # originally TFC_ORG
+    https://app.terraform.io/api/v2/organizations/"${ORG}"/subscription 2>/dev/null) # originally TFC_ORG
+
+echo "Status = " $STATUS
 
 if [ "${STATUS}" != "200" ]; then
-    echo "Failed to get organization entitlements, status ${STATUS}"
+    echo "Failed to get organization subscription, status ${STATUS}"
     if [ -f /tmp/.out.json ]; then
 	echo "Output was:"
 	cat /tmp/.out.json
 	echo "End of output"
     fi
-    fail-message "Unable to get TFC Organization Entitlements, please go back to the \"2- Terraform Cloud Trial Plan\" steps"
+    fail-message "Unable to get TFC Organization Subscriptions, please go back to the \"2- Terraform Cloud Trial Plan\" steps"
     exit 1
 fi
 
-TRIAL=$(jq -r '.data.attributes."versioned-policy-set-limit"' /tmp/.out.json 2>/dev/null)
+TRIAL=$(jq -r '.included[].attributes."identifier"' /tmp/.out.json 2>/dev/null)
+echo "############## Account identifier: " $TRIAL
 if [ -z "${TRIAL}" ]; then
-    echo ".data.attributes.versioned-policy-set-limit was empty"
+    echo ".included.[].attributes."identifier" was empty"
```

Comment:
Echo message doesn't match command. Removing the '.'

```suggestion
    echo ".included[].attributes."identifier" was empty"
```

Reply:
Also, we'll need to quote the `"identifier"` bit to make this work, e.g:

`echo ".included[].attributes.\"identifier\" was empty"`